.gitignore

```diff
@@ -133,3 +133,6 @@ packages/plans-grid-next/static/mockServiceWorker.js
 playwright-report/
 /cookies
 output/
+
```

Comment:
I don't think this change belongs to this PR :)

Reply:
Well, it could as well be,...

Reply:
Created https://github.com/Automattic/wp-calypso/pull/105901